packages/website/pages/docs/how-tos/store.md

```diff
@@ -269,6 +269,38 @@ cid, err := client.PutCar(context.Background(), carfile)
 ```
 
 </TabItem>
+<TabItem value="curl" label="HTTP API (curl)">
+
+If you're using the [HTTP API][reference-http] directly, you can upload a CAR file by sending a `POST` request to the `/car` endpoint.
+
+Your request must include an `Authorization` header with a value of `Bearer $API_TOKEN`, where `$API_TOKEN` is a valid JWT token for the Web3.Storage service. 
+
+The body of the request should be the binary CAR data, with no additional padding or encoding. Please note that the maximum size for a single CAR upload is 100 MiB. See [Working with CAR files][howto-car-files] for information about splitting CARs into smaller chunks for upload.
```

Comment:
Love this!

Is it actually 100MiB exactly ( `100 << 20` ) or is it a bit less? Since this is for developers, probably worth validating the value so folks do not get surprises...?

Reply:
That's a good call out - I know that we were having some intermittent issues with uploads that were near the limit (like 90 MiB), but that was before the new upload pipeline went live. I'll throw some test uploads at it and revise this if I hit a "soft limit".